index.js

```diff
@@ -25,9 +25,15 @@ var EDGEMAP = {
 util.inherits( FacebookInsightStream, stream.Readable )
 function FacebookInsightStream( options ) {
     stream.Readable.call( this, { objectMode: true } );
+    var isFunction = typeof options.itemList === 'function';
+    if ( !isFunction ) {
+        var itemList = options.itemList;
+        options.itemList = function () { return itemList }
```

Comment:
Would it be possible to use a different name for `itemList`? It doesn't really sound like the name of a function. Perhaps just switching it around to be `listItems`?